api/src/main/java/net/countercraft/movecraft/support/VersionInfo.java

```diff
@@ -0,0 +1,13 @@
+package net.countercraft.movecraft.support;
+
+import org.jetbrains.annotations.NotNull;
+
+public record VersionInfo(String version) {
```

Comment:
Should we call this `MinecraftVersionInfo` to reduce confusion with Movecraft versioning?

Reply:
Good point